lib/testers/rule-tester.js

```diff
@@ -138,7 +138,14 @@ const IT = Symbol("it");
  * @returns {any} Returned value of `method`.
  */
 function defaultHandler(text, method) {
-    return method.apply(this);
+    try {
+        return method.apply(this);
+    } catch (err) {
+        if (err instanceof assert.AssertionError) {
+            err.message = `${util.inspect(err.actual)} ${err.operator} ${util.inspect(err.expected)}`;
```

Comment:
Can you make this append to the existing message rather than overwriting it? I think it would be good to retain the `"Output is incorrect"` text to make the source of the error clearer.

Reply:
Oh, I forgot to explain here.
The `defaultHandler` is called nestedly somehow (sorry I don't dive into the code but its behaviour shows that), so the thrown error will be caught and err message 
 will be appended **in multiple times**. It will finally become like
```
"Output is incorrect 'a' == 'b' 'a' == 'b' 'a' == 'b'"
```
That's the reason for overwriting it here.

Another solution is adding a flag like `appended` on error object to avoid multiple appending. But is it worth to do it?

Reply:
Oh, I see. I think that's happening because `defaultHandler` is used for both the "describe" and the "it" methods. So effectively something like this is happening:

```js
defaultHandler("foo", () => {
    defaultHandler("bar", () => {
        // throw assertion error
    });
});
```

One solution would be to use the old default handler (which just calls the function in the second argument) as the default for `describe`, and use the handler that appends a diff as the default for `it`.